### ISSUE TITLE
wiki2bio/original_data/word_vocab.txt

### ISSUE THREAD

**Issue by yiyibooks** [FranxYao/dgm_latent_bow#1]
Hi,

An error was throwed when I ran the main.py: 
FileNotFoundError: [Errno 2] No such file or directory: '../wiki2bio/original_data/word_vocab.txt'

It seems that the file wiki2bio/original_data/word_vocab.txt is not included.

**Comment by FranxYao** [comment]
Oh that is for testing the model on the Wikibio data to text generation task, and not included in the paper. If you need this part could you send me an email so I can send them to you? yao.fu@columbia.edu

**Comment by yiyibooks** [comment]
Thank you very much!

In fact, I tried to Google wiki2bio later, and found this one https://github.com/tyliupku/wiki2bio.
They should be the same dataset, right?

**Comment by FranxYao** [comment]
Yep, that's right. With the current code I think you can get it run. But
during my test, the training was unstable and may corrupt in the second
epoch(loss become nan). I did not take a close look but you may figure this
out, with different initialization or smaller learning rate.
